mathml/presentation-markup/spaces/space-percentage.tentative.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>space percentages</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#space-mspace">
+<link rel="match" href="space-percentage.tentative-ref.html"/>
+<meta name="assert" content="Verify mspace visual rendering for different percentage values of height, depth and width">
+</head>
+<body>
+  <p>Test passes if you see a green square and no red.</p>
+  <div style="position: relative;">
+    <div style="position: absolute; top: 0px; left: 0px;
+                width: 200px; height: 200px;">
+       <!-- Percentage widths should be accepted -->
```

Comment:
Currently, MathML defines percentage of width as a 0: https://mathml-refresh.github.io/mathml-core/#space-mspace

It's a percentage of the default value (0px) not of the available size.